i18n/jquery.spectrum-ca.js

```diff
@@ -0,0 +1,14 @@
+// Spectrum Colorpicker
+// Catalan (ca) localization
+// https://github.com/bgrins/spectrum
+
+(function ( $ ) {
+
+    var localization = $.spectrum.localization["ca"] = {
+        cancelText: "Cancel·lar",
+        chooseText: "Escollir"
```

Comment:
Can you please add more of the keys, like: https://github.com/bgrins/spectrum/blob/master/i18n/jquery.spectrum-fr.js